docs/AboutUs.md

```diff
@@ -48,3 +48,13 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 * Role: Developer
 * Responsibilities: Logic + Testing
+
+### James Chiu
+
+<img src="images/jameschiu.jpeg" alt="jameschiu" width="200px">
```

Comment:
Minor note I think you need to rename your picture so it's `jameschiu.png` even though it isn't a png.

<img width="861" alt="Screenshot 2022-09-28 at 10 45 14 AM" src="https://user-images.githubusercontent.com/79991214/192676762-9e21c452-6fb6-4925-ac98-000797914389.png">